src/main/java/lambdastreams/task2/Book.java

```diff
@@ -0,0 +1,116 @@
+package lambdastreams.task2;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class Book {
+
+    private String name;
+    private Integer price;
+
+    public Book(String name, Integer price) {
+        this.name = name;
+        this.price = price;
+    }
+
+    public Book(String name){
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Integer getPrice() {
+        return price;
+    }
+
+    public static List<Book> createListOfBooks(Integer booksQuantity) {
```

Comment:
This method is not used anywhere - please create a test for it that will print values of this list with random books

Reply:
Created